### PR TITLE
(retroarch.c) Remove duplicate code

### PR DESCRIPTION
**Pull request by rz5**
## Description

Call a function that already exists instead of repeating the same block of code.

## Related Issues

None

## Related Pull Requests

None

## Reviewers

@twinaphex

### CODE REVIEW

File: retroarch.c [hunk 1]
```diff
@@ -21578,12 +21578,8 @@ static void video_driver_frame(const void *data, unsigned width,
    if (!video_driver_active)
       return;
 
-   if (data)
-      frame_cache_data = data;
-   frame_cache_width   = width;
-   frame_cache_height  = height;
-   frame_cache_pitch   = pitch;
-   
+   video_driver_cached_frame_set(data, width, height, pitch);
```
Comment (RobLoach) [inline retroarch.c]: Bunch of whitespace changes, but this is the change.

Comment (rz5) [reply retroarch.c]: Yeah. Sorry about the trailing whitespace removals, vscode did that for me.

Comment (RobLoach) [reply retroarch.c]: Merged master to fix whitespace changes.

File: retroarch.c [hunk 2]
```diff
@@ -28832,4 +28827,4 @@ static bool accessibility_startup_message(void)
          10);
    return true;
 }
-#endif
+#endif
```
Comment (RobLoach) [inline retroarch.c]:
``` suggestion
#endif

```

File: retroarch.c [hunk 3]
```diff
@@ -28778,7 +28774,6 @@ bool is_input_keyboard_display_on(void)
    return false;
 #endif
 }
-
 #ifdef HAVE_ACCESSIBILITY
```
Comment (RobLoach) [inline retroarch.c]:
``` suggestion

#ifdef HAVE_ACCESSIBILITY
```

**Pull request closed without merging**